### ISSUE TITLE
Completions don't work for test parameters

### ISSUE THREAD

**Issue by heejaechang** [microsoft/pylance-release#4590]
### Discussed in https://github.com/microsoft/pylance-release/discussions/4388

<div type='discussions-op-text'>

<sup>Originally posted by **rchiodo** February 17, 2023</sup>
Example code:

```python
import pytest

@pytest.fixture
def test_fixture() -> List[int]:
    return [4]

def test_func(test_fixture):
      test_fixture. # No completions here
```

We could probably implement first level completions? If the type is any and its declaration is a parameter, get the type from the extensions?</div>

**Comment by rchiodo** [comment]
> We could probably implement first level completions? If the type is any and its declaration is a parameter, get the type from the extensions?

That's what I thought too.

**Comment by heejaechang** [comment]
actual work for pyTest completion (https://github.com/microsoft/pyrx/pull/3990) is checked in but disabled by default through a setting.

we can only enable it once https://github.com/microsoft/pyrx/pull/3975 or something similar work is done

**Comment by debonte** [comment]
This issue has been fixed in prerelease version 2023.8.51, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023851-30-August-2023-PreRelease)
